client-v2/src/index.tsx

```diff
@@ -11,11 +11,19 @@ import { configReceived } from 'actions/Config';
 import { editorSetOpenFronts } from 'bundles/frontsUIBundle';
 import { storeClipboardContent } from 'actions/Clipboard';
 import { Dispatch } from 'types/Store';
+import capiQuery from 'services/capiQuery';
+import CAPIServiceContext from './components/CAPIServiceContext';
 import Modal from 'react-modal';
 
-const store = configureStore();
 const config = extractConfigFromPage();
 
+const capiServices = {
```

Comment:
To ask a possibly silly question - is this context still necessary? I can't remember if capiQuery carries state or not, but if not -- might this be better expressed as a statically imported service?

Reply:
As discussed IRL - will rip this out and have it static and ignore config option 👍